### DISCUSSION TITLE
Making HERO12/11 Black support Time Lapse long press again

### DISCUSSION THREAD

**Discussion by danielpinto29** [gopro/labs#609] [Feature Requests]
Basically Gopro Hero 11/12 [no longer supports](https://community.gopro.com/s/article/What-is-QuikCapture-and-How-Does-it-Work?language=en_US) the quickcapture feature that allows you to hold the shutter button 3 seconds to start taking photos in timelapse mode. **I get it why so many people wanted this removed but in the industry i work this feature was perfect.**
I work in the skydiving industry and tandem instructors use a [setup of 2 gopros](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQv7K9RCeJs4SzA5q0Dsx6nwkAqXaRQDFz3Og&usqp=CAU), one of them shooting video and the other one taking photos in timelapse mode. 
There should be an option to re-enable this feature again. In the rush of getting out of the airplane you dont want nor you have the time to be clicking in multiple buttons to start the camera in the adequate mode. 

Basically all i want is the good old **1 click for video, long press for timelapse photo.**

**Comment by dnewman-gpsw** [comment]
I didn't know that had changed. I checked if Labs can get the long-press message, but this behavior seems to be more core in the camera. On HERO10 the camera boots for quickcapture on the down-press of the shutter, on H11/12 the camera boots on the release of shutter, so the long-press is not known (Labs can't easily fix.)  

With Labs you can add a boot command, so that one camera defaults to timelapse photo (or interval photo on H12) and the other video. The downside is you would use the power button, not the shutter to start captures.
